specfile/specfile.py

```diff
@@ -265,4 +279,8 @@ def add_changelog_entry(
                     raise SpecfileException("Failed to auto-detect author") from e
             elif email is not None:
                 author += f" <{email}>"
-            changelog.append(ChangelogEntry.assemble(timestamp, author, entry, evr))
+            changelog.append(
+                ChangelogEntry.assemble(
+                    timestamp, author, entry, evr, append_newline=bool(changelog)
```

Comment:
Shouldn't part of this diff be in the second commit? In `Add support for %autochangelog`, there is no `append_newline` yet.

Reply:
Yes, it does, I'll fix it.